src/test/java/com/stripe/BaseStripeFunctionalTest.java

```diff
@@ -156,4 +167,49 @@ public void testMetadata(MetadataStore<?> object) throws StripeException {
 	public void testAPIBase() throws StripeException {
 		assertEquals("https://api.stripe.com", Stripe.getApiBase());
 	}
+
+	@Test
+	public void testResponseIncluded() throws
```

Comment:
@ob-stripe I was a bit confused as to where to put functional tests for this. Because it kind of covers all APIResource types but the test itself uses a Customer.

Just wanted to 2x if this is a decent spot for it as pretty much none of the other functional tests live here so it feels a bit weird.

Reply:
Ah, right. You should probably create a new `functional/ResponseTest.java` file for those two tests.